src/lib.rs

```diff
@@ -13,6 +13,103 @@
    See the License for the specific language governing permissions and
    limitations under the License.
 */
+//! ```rust,no_run
+//! // This example connects a `StackDriverExporter` to the `tracing` ecosystem. There's a lot of
+//! // pieces and API layers here, so this example is broken into _many_ steps.
+//!
+//! // The path to your Google Cloud credentials to use with this crate.
+//! let credentials_path = "my_creds.json";
+//!
+//! // The path you want to persist an OAuth2 token. Generally, the lifetime of an OAuth2 token is
+//! // about 1 hour.
+//! let persistent_token_file = Some("oauth_token.json");
+//!
+//! // The amount of time that you, the client, want to give to in-flight span export requests to
+//! // finish. If you specify `None`, then this defaults to 5 seconds.
+//! let maximum_shutdown_duration = Some(Duration::from_secs(10));
+//!
+//! // The number of span export requests that are allowed to be in-flight at once. If you specify
+//! // `None`, it defaults to 0, meaning, no limit will be enforced.
+//! let num_concurrent_requests = 2;
+//!
+//! // An implementation of `futures::task::Spawn`, which allows .
```

Comment:
This looks incomplete?

Reply:
D'oh, something I missed in the several iterations of this. This is definitely incomplete. How does the fixed-up content in 132da7bf look?